packages/language-server/src/plugins/typescript/features/CodeActionsProvider.ts

```diff
@@ -51,6 +56,27 @@ export class CodeActionsProviderImpl implements CodeActionsProvider {
             return await this.organizeImports(document, cancellationToken);
         }
 
+        if (context.only?.[0] === SORT_IMPORT_CODE_ACTION_KIND) {
+            return await this.organizeImports(
+                document,
+                cancellationToken,
+                /**skipDestructiveCodeActions */ true
+            );
+        }
+
+        // for source action command (all source.xxx)
+        // vscode would show different source code action kinds to choose from
```

Comment:
How can I trigger this code path? And what would happen then - would VS Code show a prompt of which action I prefer?

Reply:
type 'Source Action' in the command palette. This is the only command I find that is easy to debug this code action in the UI.